components/EditingLayer.svelte

```diff
@@ -23,23 +23,14 @@
       data: emptyGeojson(),
     });
 
-    $map.addLayer({
-      id: "editing-polygons",
-      source: source,
-      filter: isPolygon,
-      ...drawPolygon(color, 0.5),
-    });
-    $map.addLayer({
-      id: "editing-lines",
-      source: source,
-      filter: isLine,
-      ...drawLine(color, 1.5 * lineWidth),
-    });
+    // Don't symbolize polygons or linestrings being edited. They'll have
```

Comment:
I simplified this; it feels nicer to me now, but opinions welcome, I am far from a designer